app/representers/api/podcast_representer.rb

```diff
@@ -9,6 +9,9 @@ class Api::PodcastRepresenter < Api::BaseRepresenter
   property :updated_at
   property :published_at
 
+  property :pub_date, writeable: false
+  property :last_build_date, writeable: false
```

Comment:
probably not necessary, I guess I was thinking these are useful as they correspond to the values in the rss feed exactly